working-groups/wg-stmt-summary.md

```diff
@@ -0,0 +1,20 @@
+# Statement Summary Working Group
+
+## Goals
+
+Discussion for the designing of statement summary tables in TiDB.
+
+Now we have a statement summary table [events\_statements\_summary\_by\_digest](https://pingcap.com/docs/stable/reference/performance/statement-summary/), and more features are being added to it. But we also have some details to be discussed.
+
+## Current Issues
+
+[About table design](https://github.com/pingcap/tidb/issues/13426)
+
+## Organizer
+
+* [djshow832](https://github.com/djshow832), PingCAP
+
+## Contact
```

Comment:
It's better to follow the same structure as `wg-vec-expr` did. which is:
```
## Goals
## Meetings

// You can explain that we don't have an open meeting because there is
// only one developer in this working group. 

## Organizer
## Contact
```

Reply:
Addressed.